questions/List-manipulate.md

```diff
@@ -0,0 +1,58 @@
+# RANDLIST
+
+## Question
+
+Welcome to **RANDLIST**! Here is the problem statement:
+
+You must input a number to determine the length of a list that will contain random numbers from 1-100. The last element of the list should be removed 3 times and you must return the list to be displayed in highest value to lowest value order. 
+
+## Input
+
+
+```python
+input_len = input("What is the list length: )
+#EXAMPLE USER INPUT = 8
+
+the_list = [4,23,13,2,1,6,56,82]
+```
+
+Therefore, your function should return:
+
+```python
+[1,2,4,13,23]
+```
+
+Here are some criteria for marking someone's code:
+
+- Correctness: Does the code produce correct results for all test cases?
+- Readability: Is the code easy to read and understand?
+- Efficiency: Does the code run efficiently?
+- Style: Does the code follow good coding style and conventions?
+
+Here is a sample solution in Python:
+
+```python
+import random
+
+input_len = input("What is the list length: )
```

Comment:
```suggestion
input_len = int(input("What is the list length: "))
```
Input returns a string and we are expecting an integer so this would be a way to avoid that error.
There was a missing quotation mark near the end of the input message.